src/Expectations.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pest\Laravel;
+
+use Pest\Expectation;
+
+expect()->extend('toBeCollection', function (): Expectation {
+    // @phpstan-ignore-next-line
```

Comment:
Why this ignore? Maybe you can hint that `$this` refer to Expecation instance?

Reply:
Because the errors seen in the screenshots were occurring.

I tried adding `@var`, it didn't work.

```php
/** @var $this Expectation */
```

While I was wondering if ignoring this was the best thing to do, I saw **[this](https://github.com/lukeraymonddowning/pest-plugin-money/blob/21fbdb646da5c9d1de34917c4743064936245c61/src/Autoload.php#L23)** line and gave up looking for another solution.

Do you have a suggestion for this? @nunomaduro 

<img width="858" alt="screenshot 55" src="https://user-images.githubusercontent.com/13007665/124368034-75237e00-dc65-11eb-82a0-ab687c9b2b61.png">
<img width="622" alt="screenshot 56" src="https://user-images.githubusercontent.com/13007665/124368035-7654ab00-dc65-11eb-85a3-f85bd06b0476.png">

Reply:
It's `/** @var Expectation $this */`

Reply:
I think I posted the wrong screenshot yesterday. I tried this too. The result is the same. :/ @nunomaduro 

<img width="883" alt="screenshot 61" src="https://user-images.githubusercontent.com/13007665/124525156-d2a30080-de06-11eb-9241-ec6684680f80.png">

<img width="715" alt="screenshot 60" src="https://user-images.githubusercontent.com/13007665/124525154-d040a680-de06-11eb-87fa-c41743d20ed9.png">